plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -386,9 +387,8 @@ func (c *CiscoTelemetryMDT) handleTelemetry(data []byte) {
 			}
 		}
 
-		// if the keys and content fields are missing, skip the message as it
-		// does not have parsable data used by Telegraf
-		if keys == nil || content == nil {
+		if content == nil && !c.IncludeDeleteField {
+			c.Log.Debugf("Message skipped because no content found and include of delete field not enabled")
```

Comment:
There are models, where the `keys` can be `nil` and the `content` not. An example is `Cisco-IOS-XR-fib-common-oper:oc-aft-l3/protocol/ipv4/oor/state` (see also [here](https://www.cisco.com/c/en/us/td/docs/iosxr/ncs5500/telemetry/75x/b-telemetry-cg-ncs5500-75x/enhancememts-to-telemetry.html#Cisco_Concept.dita_7912af05-c8d8-40af-98de-c87be377e754)). 
The code panicked when only the `keys` were nil and the `content` wasn't (because you checked only if both (`keys` and `content` is `nil`)
I fixed this behavior in 9aabe02
![keys_nil](https://user-images.githubusercontent.com/6859126/206681776-aa4a89bf-28c5-415c-97eb-192b54f9de37.png)

PS: The metric is still valid because there are other tags added based on XPath, etc.